### PR TITLE
Adding README file and fixing minor errors in the steps.

### PR DESCRIPTION
**Pull request by devnook**

### CODE REVIEW

**Comment by devnook** [comment]
Reviewer: @PaulRashidi

**Comment by devnook** [comment]
CLA signed

File: polymer-build-mobile/README [hunk 1]
```diff
@@ -0,0 +1,21 @@
+This is a sample code for the Google IO 2014 codelab 
+"Polymer: Build a Material Design Mobile Web App with Paper Elements".
+
+Codelab instructions: http://io2014codelabs.appspot.com/static/codelabs/polymer-build-mobile/
+
+***
+This code sample uses Bower to manage code dependencies. Plese run
```
Comment (PaulRashidi) [inline polymer-build-mobile/README]: Can you link to somewhere that explains how to install bower?

File: polymer-build-mobile/README [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+***
+This code sample uses Bower to manage code dependencies. Plese run 
+
+    $ bower install
+
+in each directory you wish to test before running the sample in the browser.
+***
+
+***
+Polymer requires files to be served over HTTP server rather than as local files. Run
```
Comment (PaulRashidi) [inline polymer-build-mobile/README]: Perhaps define your python version requirement?

**Pull request merged**